packages/morph-attr/lib/main.js

```diff
@@ -44,10 +44,26 @@ var UNSET = { unset: true };
 
 var guid = 1;
 
-function AttrMorph(element, attrName, domHelper, namespace) {
+AttrMorph.create = function(element, attrName, domHelper, namespace) {
+  let ns = namespace !== undefined ? namespace : getAttrNamespace(attrName);
+
+  if (ns) {
```

Comment:
this branching is dubious, as it really relies on JIT heuristics to not troll us.

I can see the function being fast and inlined, then navigating to some page that has an SVG graph, and us blowing away our lovely inlined code.